package.json

```diff
@@ -47,7 +47,9 @@
     "@typescript-eslint/parser": "5.26.0",
     "eslint": "8.16.0",
     "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-prettier": "^4.0.0",
     "npm-package-json-lint": "6.3.0",
+    "prettier": "^2.6.2",
```

Comment:
remove carets

Reply:
how did this pass lint checks?

Reply:
the change in the PR broke it, but it should work now